dashboard/src/components/settings/authentication/DisableNewUsersSettings/DisableNewUsersSettings.tsx

```diff
@@ -55,47 +55,57 @@ export default function DisableNewUsersSettings() {
     throw error;
   }
 
-  const { formState, watch } = form;
-  const authDisableNewUsers = watch('authDisableNewUsers');
+  const { formState } = form;
 
   const handleDisableNewUsersChange = async (
     values: DisableNewUsersFormValues,
   ) => {
-    const updateAppMutation = updateApp({
+    const updateConfigPromise = updateConfig({
       variables: {
-        id: currentApplication.id,
-        app: {
-          ...values,
+        appId: currentApplication.id,
+        config: {
+          auth: {
+            signUp: {
+              enabled: !values.disabled,
```

Comment:
I feel we could have a more descriptive variable name than `values`.

Reply:
Ok, just saw that `values` is used a lot related forms. Maybe `formValues` or something instead? Just reading `values` and I'm a bit lost.

Reply:
Yeah, I can change that.

Reply:
I'll do this in a follow-up PR.